src/python/dxpy/scripts/dx.py

```diff
@@ -2622,19 +2622,24 @@ def list_developers(args):
     except:
         err_exit()
 
+def render_timestamp(epochSeconds):
+    # This is the format used by 'aws s3 ls'
+    return datetime.datetime.fromtimestamp(epochSeconds//1000).strftime('%Y-%m-%d %H:%M:%S')
+
 def list_database_files(args):
     try:
         results = dxpy.api.database_list_folder(
             args.database,
             input_params={"folder": args.folder, "recurse": args.recurse, "timeout": args.timeout})
-
         for r in results["results"]:
             if (args.csv == True):
+                dateStr = render_timestamp(r["modified"]) if r["modified"] != 0 else ''
```

Comment:
you can move this above condition as you need dateStr in both if and else. An I would use `date_str` as we are in python now :)

Reply:
Done